server/index.js

```diff
@@ -11,72 +11,46 @@ app.listen(PORT, () => {
   console.log('listening at port ', PORT);
 });
 
-//Tiers service
-app.use('/tiersIncluded', proxy('localhost:3101', {
-  proxyReqPathResolver: function (req) {
-    let pieces = req.url.split('?');
-    let pathname = pieces[0];
-    let queryString = pieces[1];
-    if (!queryString) {
-      return '';
-    } else {
-      return `${pathname}?${queryString}`;
-    }
-  }
-}));
-app.use('/itemsIncluded', proxy('localhost:3101', {
-  proxyReqPathResolver: function (req) {
-    let pieces = req.url.split('?');
-    let pathname = pieces[0];
-    let queryString = pieces[1];
-    if (!queryString) {
-      return '';
-    } else {
-      return `${pathname}?${queryString}`;
-    }
-  }
-}));
-
 //Bonuses service
-app.use('/bonus-track', proxy('localhost:3031', {
-  proxyReqPathResolver: function (req) {
-    let pieces = req.url.split('?');
-    let pathname = pieces[0];
-    let queryString = pieces[1];
-    if (!queryString) {
-      return '';
-    } else {
-      return `${pathname}?${queryString}`;
-    }
-  }
-}));
-app.use('/bonus-title', proxy('localhost:3031', {
-  proxyReqPathResolver: function (req) {
-    let pieces = req.url.split('?');
-    let pathname = pieces[0];
-    let queryString = pieces[1];
-    if (!queryString) {
-      return '';
-    } else {
-      return `${pathname}?${queryString}`;
-    }
-  }
-}));
-app.use('/bonus', proxy('localhost:3031', {
-  proxyReqPathResolver: function (req) {
-    let pieces = req.url.split('?');
-    let pathname = pieces[0];
-    let queryString = pieces[1];
-    if (!queryString) {
-      return '';
-    } else {
-      return `${pathname}?${queryString}`;
-    }
-  }
-}));
+// app.use('/bonus-track', proxy('http://localhost:3031', {
+//   proxyReqPathResolver: function (req) {
+//     let pieces = req.url.split('?');
+//     let pathname = pieces[0];
+//     let queryString = pieces[1];
+//     if (!queryString) {
+//       return '';
+//     } else {
+//       return `${pathname}?${queryString}`;
+//     }
+//   }
+// }));
+// app.use('/bonus-title', proxy('http://localhost:3031', {
+//   proxyReqPathResolver: function (req) {
+//     let pieces = req.url.split('?');
+//     let pathname = pieces[0];
+//     let queryString = pieces[1];
+//     if (!queryString) {
+//       return '';
+//     } else {
+//       return `${pathname}?${queryString}`;
+//     }
+//   }
+// }));
+// app.use('/bonus', proxy('http://localhost:3031', {
+//   proxyReqPathResolver: function (req) {
+//     let pieces = req.url.split('?');
+//     let pathname = pieces[0];
+//     let queryString = pieces[1];
+//     if (!queryString) {
+//       return '';
+//     } else {
+//       return `${pathname}?${queryString}`;
+//     }
+//   }
+// }));
```

Comment:
All this is preserved in your commit history. There's no need to keep this commented out code here unless you think you'll need to uncomment it in the near term.

Reply:
I probably don't need it, but since I don't have that service, I just left it for now.